bot/seasons/easter/egg_hunt/cog.py

```diff
@@ -44,14 +44,20 @@ def get_team_role(user: discord.Member) -> discord.Role:
     c.execute(f"SELECT team FROM user_scores WHERE user_id = {user.id}")
     result = c.fetchone()
     if not result:
-        new_team = random.choice([Roles.white, Roles.blurple])
-        log.debug(f"Assigned role {new_team} to {user}.")
+        c.execute(
+            "SELECT team, COUNT(*) AS count FROM user_scores "
+            "GROUP BY team ORDER BY count ASC LIMIT 1;"
+        )
+        result = c.fetchone()[0]
```

Comment:
This will fail when the hunt starts, because then the `c.fetchone()` will return None:

```py
04/21/19 12:46:27 - asyncio ERROR: Exception in callback EggHunt.task_cleanup(<Task finishe...bscriptable")>)
handle: <Handle EggHunt.task_cleanup(<Task finishe...bscriptable")>)>
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/asyncio/events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "/home/sebastiaan/pydis/seasonalbot/bot/seasons/easter/egg_hunt/cog.py", line 303, in task_cleanup
    task.result()
  File "/home/sebastiaan/pydis/seasonalbot/bot/seasons/easter/egg_hunt/cog.py", line 346, in super_egg
    count = c.fetchone()[0]
TypeError: 'NoneType' object is not subscriptable
```
```suggestion
        result = c.fetchone()
        result = result[0] if result is not None else "WHITE"
```